source/board/mkit_dk_dongle_nrf5x.c

```diff
@@ -103,6 +104,7 @@ void prerun_board_config(void)
         else if (bit2 && !bit1) {
             board_id = board_id_nrf52_dk;  // 1101
             set_target_device(1);
+            flash_manager_set_page_erase(true);
```

Comment:
This should be done for all the boards rather than just one.

Reply:
The regenerated flash algos will need to be testing before this can be merged, so that should give enough confidence to turn on page erase.